src/_posts/2019-05-10-initiative-circles.md

```diff
@@ -0,0 +1,37 @@
+
+The main way to move ICs forward are meetings. And a few considerations here: because we are a consultancy, sometimes it is difficult to find the time as you are with the client. But back to the theme of responsible adults, you can join these meetings, because they are useful for Codurance. So as long as you don’t spend all your time on them, you are free to join. Secondly, all have an equal voice here. You are the ones that decided to join and actively participate on the IC. There is no person with the higher rank on an IC. Although it can be possible that one person is designated to make sure the thing moves forward and reaches a conclusion. 
+
+And the conclusion is the last bit that I want to talk about. Sometimes an IC fails to take off the ground, and never starts. That’s fine. It wasn’t an itch worth scratching. Sometimes an IC fizzles out and never reaches a conclusion. That’s fine. Maybe the priority of the IC wasn’t that big, or maybe the effort was too much for the expected outcome. And sometimes the IC reaches a conclusion with a specific outcome. Then it gets implemented. The people actively taking part in the IC need to be in agreement. That is one of the two conditions that applies to all IC conclusions. It could be possible that people from outside the IC don’t agree, but they have had the opportunity to taking part. Also it is the responsibility of the IC members to seek advice from people impacted but they don’t need to follow everyone’s advice. The other condition is that you have done your due diligence, especially if there is a cost to it. You could have a preconceived idea about the conclusion, but is important to see as many alternatives as possible, in case you find a better outcome.
+
+And I think I mislead you talking about the previous paragraph being the last thing to talk about. Like with so many things, it requires that people participate. But, unlike in any other company I have ever worked, you can be an agent of change and improvement. Before I was reduced to complain about the situation, now I can be the change. If I don’t, well, can’t complain, can I?
```

Comment:
Perhaps "Before I was reduced to complain about the situation" -> "Before I could only complain about bad situations"